packages/block-library/src/navigation/editor.scss

```diff
@@ -61,11 +61,17 @@
 	// Polish the Appender.
 	.wp-block-navigation .block-list-appender {
 		margin: 0;
+		width: 28px;
+		height: 28px;
+		margin-top: $grid-size/2;
```

Comment:
I think you could use `$grid-size-small` instead of dividing by 2.

Reply:
Yes, you are right!